mlflow/models/__init__.py

```diff
@@ -64,6 +64,8 @@ def log(cls, artifact_path, flavor, registered_model_name=None, **kwargs):
         """
         Log model using supplied flavor module.
 
+        Note: If no run is active, it will instantiate a run to obtain a run_id.
```

Comment:
```suggestion
        Note: If no run is active, this method will create a new active run.
```

Just a small wording suggestion, lmk what you think (want to make it clear that we set the "active" run)

Reply:
I agree, I think the suggestion is clearer.